src/components/AvatarGroup/AvatarGroupCounterTooltipContentVirtualizedList.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { Ref, useMemo } from "react";
+import VirtualizedList from "../VirtualizedList/VirtualizedList";
+import VibeComponentProps from "../../types/VibeComponentProps";
+import { AvatarTypes } from "../Avatar/AvatarConstants";
+import { AvatarProps } from "../Avatar/Avatar";
+import styles from "./AvatarGroupCounterTooltipContentVirtualizedList.module.scss";
+
+const LIST_OPTIONS = Object.freeze({
+  maxItemsWithoutScroll: 10,
+  itemLineHeight: 34,
+  itemLineWidth: 150
+});
+
+interface AvatarGroupCounterTooltipContentVirtualizedListProps extends VibeComponentProps {
+  /**
+   * Array of Avatar components
+   */
+  avatarItems?: { value: AvatarProps & { tooltipContent: any } }[];
+  avatarRenderer?: (
+    item: { value: AvatarProps & { tooltipContent: any } },
+    index: number,
+    style: any,
```

Comment:
There is a type for style, you can find examples in other converted files :)

Reply:
Fixed